delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -231,60 +180,59 @@ def checkoutSCM(repo_name) {
 	dir(repo_name)
 	{
 		def repo_url = scmModule.getRepoCloneUrl(repo_name)
-		//events.sendStartedEvent('GET_SERVICE_CODE')
 		try {
 			checkout([$class: 'GitSCM', branches: [
 				[name: '*/master']
 			], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [
 				[credentialsId: configLoader.REPOSITORY.CREDENTIAL_ID, url: repo_url]
 			]])
 		}catch(ex) {
-			updateServiceInDB(var_db_service_id, configLoader.AWS.API.PROD_ID, configLoader.AWS.REGION, "deletion_failed");
 			send_status_email ("FAILED")
+			events.sendFailureEvent('DELETE_PROJECT', ex.getMessage(), context_map)
 			error "checkoutSCM Failed. "+ex.getMessage()
 		}
 	}
 }
 
 /**
- * Update the service name in serverless config file
+ * Update the service name in serverless service_config file
  * @param  domain
  * @param  serviceName
  * @return
  */
-def updateServiceNameConfig(envval,config) {
+def updateServiceNameConfig(envval) {
```

Comment:
looks like the parameter `envval` isn't required. Can we remove it?

Reply:
Removed the unwanted parameter